docs/manual/howtos/qos.rst

```diff
@@ -89,42 +89,46 @@ The entity-specific QoSes can be found in the `qos` sub-namespace of the same na
 	| TypeConsistencyEnforcement | N                    | N            | N             | Y        | Y             | Y             |
 	+----------------------------+----------------------+--------------+---------------+----------+---------------+---------------+
 
-Setting of QoSPolicies can be done either through left-shifting the QoSPolicy "into" the QoS:
+Setting of QoSPolicies can be done by:
+
+Either left-shifting the QoSPolicy "into" the QoS:
 
 .. code:: C++
 
 	dds::sub::qos::DataReaderQos rqos;
 	rqos << dds::core::policy::Durability(dds::core::policy::DurabilityKind::TRANSIENT_LOCAL);
 
-, or passing it as the parameter of the `policy` function:
+Or passing it as the parameter of the `policy` function:
 
 .. code:: C++
 
 	dds::pub::qos::DataWriterQos wqos;
 	dds::core::policy::Reliability rel(dds::core::policy::ReliabilityKind::RELIABLE, dds::core::Duration(8, 8));
 	wqos.policy(rel);
 
-Whereas getting of QoSPolicies can be done either through the right-shifting the QoSPolicy "out of" the QoS:
+Getting of QoSPolicies can be done by:
+
+Either through the right-shifting the QoSPolicy "out of" the QoS:
 
 .. code:: C++
 
 	dds::topic::qos::TopicQos tqos;
 	dds::core::policy::TopicData td;
 	tqos >> td;
 
-, or through the `policy` function, which is templated to indicate which QoSPolicy is being accessed:
+Or through the `policy` function, which is templated to indicate which QoSPolicy is being accessed:
 
 .. code:: C++
 
 	dds::domain::qos::DomainParticipantQos dqos;
 	auto ud = dqos.policy<dds::core::policy::UserData>();
 
-For a more detailed explanation of the different QoSPolicies and their effects on the behaviour of CycloneDDS, the user is referred to the OMG DDS Spec v1.4 section 2.2.3.
+For a detailed explanation of the different QoSPolicies and their effects on the behaviour of CycloneDDS, refer to the OMG DDS Spec v1.4 section 2.2.3.
 
 Default and Inherited QoSes
 ---------------------------
 
-QoSes have a number of default settings that are falled back to when none are provided upon creation.
+QoSes have a number of default settings that are falled-back to when none are provided on creation.
```

Comment:
Error on my part, the past tense of "fall back" is "fell back"

Reply:
use "reverted to"?